### ISSUE TITLE
Cut new release: `0.1.4`

### ISSUE THREAD

**Issue by jaimergp** [napari/napari-plugin-manager#122]
Once https://github.com/napari/napari-plugin-manager/pull/121 is merged.

I don't know if we need `0.1.4` or `0.2.0`. I think we had API changes, right @dalthviz? In that case, we should go for `0.2.0`. [BUT at the same time `napari` is pinning minor versions](https://github.com/napari/packaging/blob/205713d2e28b39d8dcebbebe0666d1432549a48e/conda-recipe/meta.yaml#L121), so maybe it's just `0.1.4` after all.

**Comment by dalthviz** [comment]
Doing a release make sense to me :+1: Regarding the version, technically I would say `0.2.0` should be used since now a kind of API exists indeed. However, the package is just being used by napari and also, discussing with Jaime, still a major change needs to be done: separating the code into two actual different packages (`plugin-manager` and `napari-plugin-manager`). That change will require at least a minor release so maybe using version `0.2.0` is better suited for that :thinking:

What do you think @psobolewskiPhD @Czaki @jni ? Is it better to do a new release using `0.1.4` or `0.2.0` as version? Let us know!

**Comment by Czaki** [comment]
Does this API changes will break existing napari releases?

**Comment by dalthviz** [comment]
Nope, I don't think so. In fact, the way the dialog should be imported from napari remains being the same even with the new base classes definitions

**Comment by Czaki** [comment]
So if not break older napari then no need to bump (or you know some other tool that is using it?).

**Comment by dalthviz** [comment]
I don't think any other tool is using it indeed :+1: So going with 0.1.4 when doing the release for the moment. Thanks for the input!

**Comment by dalthviz** [comment]
Hi there, created a v0.1.4 release:

* GitHub release: https://github.com/napari/napari-plugin-manager/releases/tag/v0.1.4
* PyPI: https://pypi.org/project/napari-plugin-manager/0.1.4/
* conda-forge related issue/PR to update recipe (PR needs approval and merge): https://github.com/conda-forge/napari-plugin-manager-feedstock/issues/12 and https://github.com/conda-forge/napari-plugin-manager-feedstock/pull/13

Let me know if something else is needed!
